Source/JavaScriptCore/wasm/WasmLLIntGenerator.cpp

```diff
@@ -1175,6 +1204,21 @@ auto LLIntGenerator::addTry(BlockSignature signature, Stack& enclosingStack, Con
     return { };
 }
 
+auto LLIntGenerator::addTryTable(BlockSignature signature, Stack& enclosingStack, const Vector<CatchHandler>& targets, ControlType& result, Stack& newStack) -> PartialResult
+{
+    m_usesExceptions = true;
+    ++m_tryDepth;
+
+    splitStack(signature, enclosingStack, newStack);
+
+    Ref<Label> tryLabel = newEmittedLabel();
+    Ref<Label> continuation = newLabel();
+    result = ControlType::createTryTable(signature, m_stackSize, WTFMove(tryLabel), WTFMove(continuation), m_tryDepth);
+    m_pendingTryTables.add(&std::get<ControlTryTable>(result).m_try.get(), targets);
```

Comment:
Let's put the table into corresponding `ControlType` instead of using `m_pendingTryTables`.